pgc_ndvi.py

```diff
@@ -54,7 +54,9 @@ def main():
                              "default is slurm_ndvi.py, in script root folder)")
     parser.add_argument("-l", help="PBS resources requested (mimicks qsub syntax, PBS only)")
     parser.add_argument("--log", nargs='?', const="default",
-                        help="path to file to log progress (default is ortho_<timestamp>.log next to the <dst dir>")
+                        help="output log file -- top level log is not written without this arg. "
```

Comment:
Perf. I like the default behavior not requiring an argument for this option.  Cool.